### PR TITLE
allow identity scheduler

### PR DESCRIPTION
**Pull request by kmdalton**
The control flow in `pytorch_fob/optimizers/lr_schedulers/__init__.py:get_lr_scheduler` was blocking me from setting `scheduler: identity`. I think this PR sets it back to how it is supposed to function. Of course, I could be missing something here. I'm still getting familiar with the code base. 

Thanks for this project!

### CODE REVIEW

**Comment by simonblauth** [comment]
Glad you like it!
You are correct, I had fixed this in aaddca529cf5116867576a484b1035acbef83291, however only on the on the [dev](https://github.com/automl/FOB/tree/dev) branch. Ill merge your PR for now until i get around to merging the dev branch.

**Comment by kmdalton** [comment]
@simonblauth, would you recommend that I use the dev branch to test my optimizer? Or should I stick with main? Thanks!

**Comment by simonblauth** [comment]
Well its always a tradeoff, dev branch has more recent features (and sometimes bugfixes). However, most changes are not yet documented in the readmes and such, so there might be some mismatches. Overall I'd recommend sticking to the main branch as most features added on dev are not really obvious to use yet. But I myself always work on dev, so its not unstable or anything.
